torch/csrc/jit/tensorexpr/tensorexpr_tutorical.py

```diff
@@ -0,0 +1,219 @@
+# Tensor Expression: python API
+#
+# This tutorial explains the python API of TE that includes: 1) TE IR, 2) TE
+# loop transformations, and 3) TE codegen.
+
+import torch
+import torch._C._te as te
+from numpy import median
+import timeit
+
+LLVM_ENABLED = torch._C._llvm_enabled()
+
+# 1. End-to-End example: addition of two tensors
+
+# We first create two BufHandles in TE which correspond to the two input
+# tensors.
+
+# The data (physical memory) of a tensor is represented as a buffer
+# in TE, which has dimensions, a data type and a name. BufHandle is a pointer
+# to a buffer that defines load/store operations on it. Here, A points to a
+# buffer with dimentions [32], 'float' data type and name 'A'; B points to a
+# buffer with dimentions [32], 'float' data type and name 'B'.
```

Comment:
It might be worth emphasizing that `Buffer` and `BufHandle` do not actually point to any real data, they just denote the corresponding symbols and specify properties of the data.